crates/sparrow-main/tests/e2e/fixture/parquet_table_builder.rs

```diff
@@ -79,6 +80,11 @@ impl ParquetTableBuilder {
             parquet_writer.close().unwrap();
         }
 
+        // The file modification time is used as the prepare hash for local files.
+        // For determinism, we set this to a fixed value in tests.
+        filetime::set_file_mtime(temp_file.path(), PARQUET_FILE_TIME)
```

Comment:
Since systems that don't provide etags rely on the "last modified time", updated this to generate parquet files with a specific modification time.